common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -328,23 +340,27 @@ object DotcomRenderingDataModel {
     }
 
     val calloutsUrl: Option[String] = combinedConfig.fields.toList
-      .filter(entry => entry._1 == "calloutsUrl")
-      .headOption
+      .find(_._1 == "calloutsUrl")
       .flatMap(entry => entry._2.asOpt[String])
 
+    def toDCRBlock = { block: APIBlock =>
+      Block(block, page, shouldAddAffiliateLinks, request, false, calloutsUrl, contentDateTimes)
+    }
+
     val mainMediaElements =
       mainBlock
-        .map(block => Block(block, page, shouldAddAffiliateLinks, request, true, calloutsUrl, contentDateTimes))
+        .map(toDCRBlock)
```

Comment:
In the `toDCRBlock` abstraction above we're now hardcoding one of the parameters to `false` but in the original code it was true for `mainBlock` and only false for `bodyBlocks`. Just saying.

Reply:
good spot, I'll fix